app/models/rent.rb

```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+  validates :user, :book, :from, :to, presence: true
```

Comment:
In order to check for book presence you can use: `belongs_to :book, required: true`